src/web/app/pages-instructor/instructor-courses-page/instructor-courses-page.component.ts

```diff
@@ -350,47 +385,31 @@ export class InstructorCoursesPageComponent implements OnInit {
 
       isClosingEmailEnabled: fromFeedbackSession.isClosingEmailEnabled,
       isPublishedEmailEnabled: fromFeedbackSession.isPublishedEmailEnabled,
-    }).pipe(
-      switchMap((feedbackSession: FeedbackSession) => {
-        createdFeedbackSession = feedbackSession;
-        return this.feedbackQuestionsService.getFeedbackQuestions({
-          courseId: fromFeedbackSession.courseId,
-          feedbackSessionName: fromFeedbackSession.feedbackSessionName,
-          intent: Intent.FULL_DETAIL,
-        });
-      }),
-      switchMap((response: FeedbackQuestions) => {
-        if (response.questions.length === 0) {
-          // no questions to copy
-          return of(createdFeedbackSession);
-        }
-        return from(response.questions).pipe(
-          concatMap((feedbackQuestion: FeedbackQuestion) => {
-            return this.feedbackQuestionsService.createFeedbackQuestion(
-              createdFeedbackSession.courseId, createdFeedbackSession.feedbackSessionName, {
-                questionNumber: feedbackQuestion.questionNumber,
-                questionBrief: feedbackQuestion.questionBrief,
-                questionDescription: feedbackQuestion.questionDescription,
-
-                questionDetails: feedbackQuestion.questionDetails,
-                questionType: feedbackQuestion.questionType,
-
-                giverType: feedbackQuestion.giverType,
-                recipientType: feedbackQuestion.recipientType,
-
-                numberOfEntitiesToGiveFeedbackToSetting: feedbackQuestion.numberOfEntitiesToGiveFeedbackToSetting,
-                customNumberOfEntitiesToGiveFeedbackTo: feedbackQuestion.customNumberOfEntitiesToGiveFeedbackTo,
-
-                showResponsesTo: feedbackQuestion.showResponsesTo,
-                showGiverNameTo: feedbackQuestion.showGiverNameTo,
-                showRecipientNameTo: feedbackQuestion.showRecipientNameTo,
-              });
-          }),
-          last(),
-          switchMap(() => of(createdFeedbackSession)),
-        );
-      }),
-    );
+    };
+  }
+
+  /**
+   * Creates a FeedbackQuestionCreateRequest by cloning the given feedback question.
+   */
+  private fbQuestionCreationReq(feedbackQuestion: FeedbackQuestion): FeedbackQuestionCreateRequest {
```

Comment:
`toFbQuestionCreationReq` or `newFbQuestionCreationReq`? Making it an action sounding name is probably better.